src/hooks/useMediaQuery.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
```

Comment:
```suggestion
import { useState, useEffect } from 'react';
```

import * as 방식은 모듈 크기가 증가하고 명확하지 않은 것 같습니다.
리액트 훅들은 필요한 부분만 명시적으로 가져오는 것이 좋을 것 같아요!

Reply:
근데 지금 보니까 shadcn에서 제공해주는 코드는 싹다 `import * as React from 'react';` 이네요.. 저희가 추가하는 것들이라도 명시적으로 import 하는 건 어떠세요?

Reply:
shadcn 제공 코드도 다 수정하는 방향을 말씀하신 걸까요?